dali/test/python/test_operator_random_resized_crop.py

```diff
@@ -117,4 +137,6 @@ def test_random_resized_crop():
                     ((1, 2), (0.4, 1.0)),
                     ((0.5, 1), (0.1, 0.5))]:
                     for size in [(100,100), (640,480)]:
-                        yield _test_rrc, device, max_frames, layout, aspect, area, size
+                        input_type = types[np.random.randint(0, len(types))]
```

Comment:
Can we just create a separate, deterministic test that would just go though all data types, and set other test parameters to one, well defined value?
On the other hand the set of input types is rather very immitted, so I think we will always test it anyway.

Reply:
I don't wont to make this test run any longer than it absolutely needs to. There are just 6 options (3 input types + output original vs float). The test is deterministic (fixed seed) and all combinations are tested.

Reply:
OK